lib/responsys/i18n/en.yml

```diff
@@ -18,41 +22,6 @@ en:
           empty_event: The event_name or event_id must be specified
           incorrect_event_object: "custom_event must be a CustomEvent instance"
       client:
-        available_methods:
-          - wsdl
-          - endpoint
-          - namespace
-          - raise_errors
-          - proxy
-          - headers
-          - open_timeout
-          - read_timeout
-          - ssl_verify_mode
-          - ssl_version
-          - ssl_cert_file
-          - ssl_cert_key_file
-          - ssl_ca_cert_file
-          - ssl_cert_key_password
-          - convert_request_keys_to
-          - soap_header
-          - element_form_default
-          - env_namespace
-          - namespace_identifier
-          - namespaces
-          - encoding
-          - soap_version
-          - basic_auth
-          - digest_auth
-          - wsse_auth
-          - wsse_timestamp
-          - ntlm
-          - strip_namespaces
-          - convert_response_tags_to
-          - logger
-          - log_level
-          - log
-          - filters
-          - pretty_print_xml
```

Comment:
I knoww It was not used anywhere in the code anymore. I think it was used in the deleted code.

Also I don't believe we should maintain and filter the attributes you can pass to Savon, it's the user's fault if his params are wrong.
Savon will also be removed in a newer release if we go RESTing too. :godmode: